horace_core/sqw/page_operations/PageOp_apply.m

```diff
@@ -53,19 +55,17 @@
             if obj.changes_pix_only
                 return;
             end
-            new_signal     = page_pxls.signal;           
+            new_signal     = page_pxls.signal;
```

Comment:
Why have you swapped abbreviation? `pix` everywhere else.

Reply:
its `page_pix` property, so `page_pix` -- local variable. but MATLAB does not like variable looking like property so `page_pxls`

Reply:
This is inconsistent with the rest of Horace. If you have to, call it `local_page_pix`, but don't swap abbreviations like this.

Reply:
do not see any problem in naming local variable with sope of one screen any way I like. It is meaningful anyway and have no relations to any other local variables in Horace.

Reply:
The point is that it is not just you who will have to maintain this code in future and even small changes to consistency like this cause other to have to think twice when they're making changes.

Reply:
I here mean anybody. This is local variable with scope of 20 rows. Its meaning is clean so I do not see problem of it being anything.